tsl/test/sql/include/cagg_on_cagg_common.sql

```diff
@@ -2,6 +2,8 @@
 -- Please see the included NOTICE for copyright information and
 -- LICENSE-TIMESCALE for a copy of the license.
 
+\set VERBOSITY default
```

Comment:
why is this needed?

Reply:
want to see some `errdetail` in test output, specially in validations

Reply:
Second that, it seems excessive. See previous comments.

Reply:
Fixed!